fdbserver/workloads/IndexPrefetchDemo.actor.cpp

```diff
@@ -0,0 +1,171 @@
+/*
+ * IndexPrefetchDemo.actor.cpp
```

Comment:
it means you can run this in multi-tester for perf tests

Reply:
I don't understand. Could you elaborate it?

Reply:
multi-tester is a load generator framework FDB uses to generate load.
Each fdbserver can start as a tester role, which will pick up the specified `workloads`, like the one you wrote. 
These testers will behave as "clients generate traffic to the test cluster" .

Let's sync offline on how to set it up in our internal infra.